benchmark/cmake/modules/AddSwiftBenchmarkSuite.cmake

```diff
@@ -20,6 +20,58 @@ function(runcmd)
   set(${RUNCMD_VARIABLE} ${${RUNCMD_VARIABLE}} PARENT_SCOPE)
 endfunction(runcmd)
 
+function (swift_benchmark_library objfileout sibfileout)
+  cmake_parse_arguments(BENCHLIB "" "MODULE_PATH;SOURCE_DIR;OBJECT_DIR" "SOURCES;LIBRARY_FLAGS" ${ARGN})
```

Comment:
If I'm not mistaken `cmake_parse_arguments` accepts a prefix then three lists for the option flags, single-value options and multi-value options, then the arguments to parse. Looks like you need to add another empty list before `${ARGN}`.